tests/accumulate-headers-after-304.js

```diff
@@ -0,0 +1,212 @@
+// Daft-based Squid Tests
+// Copyright (C) The Measurement Factory.
+// http://www.measurement-factory.com/
+
+// Tests whether an HTTP proxy does not allow to increase an HTTP response header
+// via 304 responses more than the specified limit.
+
+import HttpTestCase from "../src/test/HttpCase";
+import Field from "../src/http/Field";
+import Body from "../src/http/Body";
+import Resource from "../src/anyp/Resource";
+import * as ConfigurationGenerator from "../src/test/ConfigGen";
+import * as AddressPool from "../src/misc/AddressPool";
+import * as Http from "../src/http/Gadgets";
+import * as FuzzyTime from "../src/misc/FuzzyTime";
+import * as Gadgets from "../src/misc/Gadgets";
+import * as Config from "../src/misc/Config";
+import assert from "assert";
+import Test from "../src/overlord/Test";
+
+const DefaultPrefixSizeKB = 64;
+const DefaultPrefixSize = DefaultPrefixSizeKB*1024;
+
+Config.Recognize([
+    {
+        option: "smp",
+        type: "Boolean",
+        default: "false",
+        description: "In this mode MISS, UPDATE and HIT requests will go to different proxy SMP workers",
+    },
```

Comment:
Please use `poke-same-worker` from squid-dafts commit 14b2e3b instead of `smp`. Technically, Squid instances with a rock cache_dir are using SMP mode (by default), even if they have one worker, so we do not want to use that "SMP" terminology in this context.

Reply:
Done.